sdk/template/azure-sdk-template/README-EXAMPLE.md

```diff
@@ -0,0 +1,125 @@
+# Azure Template client library for Java
+
+This is a template README for Azure Java libraries, it should be used as a starting point for new libraries added into this repository.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation]| [Product documentation][search_docs] | [Samples][samples]
```

Comment:
Given the number of links the guidance has changed to make these a list instead of a single line.  See https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-EXAMPLE.md

Reply:
Didn't realize this, will update this, the Java READMEs likely need to be updated to follow that guideline.